src/sidebar/test/oauth-auth-test.js

```diff
@@ -1,76 +1,23 @@
 'use strict';
```

Comment:
Most of the changes in this file are removing stubbing of HTTP requests/responses and authorization window opening and replacing them with calls to a fake `OAuthClient` instance.